examples/verify-contract/index.js

```diff
@@ -32,7 +32,7 @@ async function main() {
     solidityFilePath: 'contracts/Vault.sol',
     contractName: 'VaultV2',
     contractAddress: '0x38e373CC414e90dDec45cf7166d497409902e998',
-    contractNetwork: 'goerli',
+    contractNetwork: 'sepolia',
```

Comment:
We should also change the contract address if we change the network, we need to keep the same contract ABI